cpp/include/cudf/wrappers/durations.hpp

```diff
@@ -33,6 +33,14 @@ namespace cudf {
  * @brief Type alias representing an int32_t duration of days.
  */
 using duration_D = cuda::std::chrono::duration<int32_t, cuda::std::chrono::days::period>;
+/**
+ * @brief Type alias representing an int32_t duration of hours.
+ */
+using duration_h = cuda::std::chrono::duration<int32_t, cuda::std::chrono::hours::period>;
+/**
+ * @brief Type alias representing an int32_t duration of minutes.
+ */
+using duration_m = cuda::std::chrono::duration<int32_t, cuda::std::chrono::minutes::period>;
```

Comment:
Since we have `duration_D` (capital `D`), I wonder if we should have `duration_H` and `duration_M` for consistency.

Reply:
I agree maybe using capital makes sense, but I don't think its in the scope of this PR as there are other places where this will likely need to be changed?

Reply:
Why it's not in scope of this PR? You introduced those aliases (`duration_h`, `duration_m`) in this PR, didn't you?

Reply:
right, but I was following the convention in `timestamps.hpp`, within it, `timestamp_D` had a capitalized last letter but all other resolutions were lower case.

Reply:
I believe that the convention in `timestamps.hpp` is using big cap for Day and small cap from second to millisecond to nanosecond etc. @mythrocks?

Reply:
In addition, `duration_m` maybe confused whether it is "month" or "minute"?